src/vivarium/framework/engine.py

```diff
@@ -241,7 +252,7 @@ def setup(self) -> None:
 
         post_setup = self._builder.event.get_emitter("post_setup")
         self._lifecycle.set_state("post_setup")
-        post_setup(None)
+        post_setup(pd.Index([]), None)
```

Comment:
For posterity, it's because this callable is `EventChannel.emit()` which takes two arguments - a pd.Index and an optional dictionary, with a default value of None.

This change is effectively changing the first (pd.Index) argument from None to an empty index, which is fine, because `post_setup` methods won't use the index anyway as the simulants haven't yet been initialized.